lib/v2/ikea/cn/family_offers.js

```diff
@@ -0,0 +1,34 @@
+const got = require('@/utils/got');
+const { generateRequestHeaders, generateProductItem } = require('./utils');
+
+module.exports = async (ctx) => {
+    const api = 'https://srv.app.ikea.cn/content/recommendation/v2/product-group/products';
+    const link = 'https://www.ikea.cn/cn/zh/offers/family-offers/';
+
+    const apiRequestParams = {
+        processOutOfStock: 'SORT',
+        groupId: 'cms_product_cn--zh--23fe3210a3e211ebbf58a38a02d586b9_2_0',
+        page: 1,
+        size: 200,
+    };
+
+    const apiSearchParams = new URLSearchParams();
+    for (const key in apiRequestParams) {
+        apiSearchParams.append(key, apiRequestParams[key]);
+    }
+
+    const url = `${api}?${apiSearchParams.toString()}`;
+    const headers = generateRequestHeaders();
+
+    const response = await got({
+        url,
+        headers,
+    });
```

Comment:
You can pass the whole search params to got
```js
const response = await got({
        url,
        headers,
        searchParams: apiSearchParams,
    });
```
No need to append the search params to url string